conversions/Forwards/src/main/java/com/agentdid127/resourcepack/forwards/ForwardsPackConverter.java

```diff
@@ -32,6 +32,7 @@ public ForwardsPackConverter(String from, String to, String light, boolean minif
         converterRunner(from, to, light);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Should not suppress deprecation warnings. The current warning is for a converter which may be removed.